ale_linters/ansible/ansible_lint.vim

```diff
@@ -18,28 +18,16 @@ function! ale_linters#ansible#ansible_lint#Handle(buffer, lines) abort
         endif
     endfor
 
-    " Matches patterns line the following:
-    "
-    " test.yml:35: [EANSIBLE0002] Trailing whitespace
```

Comment:
Keep an example here in the comments showing what the pattern is trying to match.

Reply:
Examples are below.